### PR TITLE
Release preparation

### PR DESCRIPTION
**Pull request by frederike-ramin**

### CODE REVIEW

File: src/main/java/de/hpi/bpt/argos/core/Application.java [hunk 1]
```diff
@@ -30,8 +30,8 @@ public static void main(String[] args) {
 
 			PropertyEditorImpl.getInstance().setProperty(splitArg[0], splitArg[1]);
 		}
-		(new ArgosImpl()).start();
 
+		Argos argos = ArgosImpl.run();
```
Comment (hpibptlab) [inline src/main/java/de/hpi/bpt/argos/core/Application.java:34]: MAJOR Remove this unused "argos" local variable. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1481)
MAJOR Remove this useless assignment to local variable "argos". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1854)

File: src/main/java/de/hpi/bpt/argos/util/threading/BackgroundWorkerImpl.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+/**
+ * {@inheritDoc}
+ * This is the implementation.
+ */
+public class BackgroundWorkerImpl<DataType> implements BackgroundWorker<DataType> {
+	private static final Logger logger = LoggerFactory.getLogger(BackgroundWorkerImpl.class);
```
Comment (hpibptlab) [inline src/main/java/de/hpi/bpt/argos/util/threading/BackgroundWorkerImpl.java]: MAJOR Remove this unused "logger" private field. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1068)

**Comment by hpibptlab** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Review by tommartensen — APPROVED** [review]

File: src/main/java/de/hpi/bpt/argos/util/threading/BackgroundWorkerImpl.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
+package de.hpi.bpt.argos.util.threading;
+
+import org.slf4j.Logger;
```
Comment (hpibptlab) [inline src/main/java/de/hpi/bpt/argos/util/threading/BackgroundWorkerImpl.java]: MINOR Remove this unused import 'org.slf4j.Logger'. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/de/hpi/bpt/argos/util/threading/BackgroundWorkerImpl.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+import org.slf4j.LoggerFactory;
```
Comment (hpibptlab) [inline src/main/java/de/hpi/bpt/argos/util/threading/BackgroundWorkerImpl.java]: MINOR Remove this unused import 'org.slf4j.LoggerFactory'. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by hpibptlab** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by hpibptlab** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

**Pull request merged**